src/app/questions/file/file.component.ts

```diff
@@ -113,4 +125,30 @@ export class FileComponent implements ControlValueAccessor, OnInit {
 
   }
 
+  // adding save values to from control
+  private _showSavedAnswers() {
+    if ((this.reviewStatus === 'in progress') && (this.doReview)) {
+      if (!this.innerValue) {
+        this.innerValue = {
+          answer: [],
+          comment: ''
+        };
+      }
+      if (this.review.comment) {
+        this.innerValue.comment = this.review.comment;
+        this.comment = this.review.comment;
+      }
+      if (this.review.answer) {
+        this.innerValue.answer = this.review.answer;
+      }
+    }
+    if ((this.submissionStatus === 'in progress') && (this.doAssessment)) {
+      if (!this.innerValue) {
+        this.innerValue = [];
+      }
+      this.innerValue = this.submission.answer;
```

Comment:
Since we are assigning value to `innerValue` here, don't need the initialisation